mobile/src/libs/components/card/card.tsx

```diff
@@ -28,28 +31,45 @@ const Card: React.FC<Properties> = ({
   iconName,
   iconColor,
   onPress,
+  onDelete,
 }) => {
   const handlePress = (): void => {
     onPress(title);
   };
 
+  const handleDelete = (): void => {
+    if (onDelete) {
+      onDelete();
+    }
+  };
+
+  const rightSwipeActions = (): ReactNode => {
```

Comment:
```suggestion
  const rightSwipeActions = (): React.ReactNode => {
```

Reply:
i have imported type ReactNode, so i guess its not necessary)

Reply:
Is it not a similar situation as React.FC/etc.? To keep the same import styling throughout the project?

Reply:
@OlesiaTr i remember we have always write like type ReactNode, but if its critical now, ill change it )